Unaccountable-Magpie/BangazonSite
# ProductViews index displaying when clicking the navbar link

### Purpose:

Got ProductViews index to display when clicking the navbar link

### How it fits in context of the project:

https://github.com/Unaccountable-Magpie/BangazonSite/issues/39


### Specific Feature affected:

Added ProductTypesViewModel,ProductTypesController, and View Model

### How to test (Be thorough!):

1.git fetch -a
2. git checkout evStarter
3.start Bangazon.sln
4. Click ISS Express GREEN button
5. Click Product Types in the NavBar
6. Index should show -- a list including Clothing and Accessories
7. Click on Clothing, should bring up a detail page
